bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieDeferredSyncTest.java

```diff
@@ -121,7 +121,7 @@ private void testClose(boolean force) throws Exception {
                     .withLedgerId(ledgerId)
                     .withRecovery(true)
                     .withPassword(new byte[0])
-                    .execute());) {
+                    .execute())) {
```

Comment:
why checkstyle doesn't catch this?

Reply:
Currently, there is no check defined to catch this violation. Since the [8.22 version](https://checkstyle.sourceforge.io/releasenotes.html#Release_8.22) of checkstyle it is possible to add the `UnnecessarySemicolonInTryWithResources` module which should prevent this type of violations. I will try to update the version of checkstyle and add this module to the checkstyle.xml in a separate pull request.

Reply:
Currently, there is no check defined to catch this violation. Since the [8.22 version](https://checkstyle.sourceforge.io/releasenotes.html#Release_8.22) of checkstyle it is possible to add the `UnnecessarySemicolonInTryWithResources` module which should prevent this type of violations. I will try to update the version of checkstyle and add this module to the checkstyle.xml in a separate pull request.